docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -87,8 +87,18 @@ \section{Failure Mode and Effect Analysis}
 
 \section{Safety and Security Requirements}
 
-\wss{Newly discovered requirements.  These should also be added to the SRS.  (A
-rationale design process how and why to fake it.)}
+\subsection*{Safety Requirements}
+\begin{enumerate}
+  \item[SFR-1.] Graphics shall avoid changing of brightness at rapid rate to take account for users prone to seizures.\\
+  \item[SFR-2.] Colours should avoid brightness that can be damaging to users' eyes.\\
+  \item[SFR-3.] Untrained users should not need to interact with any electronic equipment to avoid potential injury.\\
+\end{enumerate}
+
+\subsection*{Security Requirements}
+\begin{enumerate}
+  \item[SCR-1.] Interface shall prevent any modifications or injections of measurements from the user.
+  \item[SCR-2.] Only authorized users are allowed to mnodify concealed calculations, settings and/or parameters.
+\end{enumerate}
```

Comment:
Maybe we can add a couple more so I can discuss it some more in the road map.

Also there is a typo "modify" (line 100)